packages/vesting-base-lp/src/handlers.rs

```diff
@@ -340,6 +340,7 @@ fn execute_migrate_liquidity(
     };
 
     if max_available_amount.is_zero() {
+        MIGRATION_STATUS.save(deps.storage, &MigrationState::Completed)?;
```

Comment:
I've been thinking and I can see that if we stick with this approach we leave the empty accounts outdated, e.g. we have 10 vesting accounts, we migrate in batches of 4 and 2 last accounts are empty:
- migrate 1-4 accounts: migrated completely;
- migrate 5-8 accounts: migrated completely;
- migrate 9-10 accounts:  `max_available_amount == zero` => leave at this code line and skip the `post_migration_vesting_reschedule_callback` where we recalculate available/total amounts.

As far as I can see we are safe to delete lines 342-345 at all and process the whole migration flow for all users. As the result there won't be any users left with their vesting info not updated with the new values. @swelf19 please take a look and share your thoughts on it 🙏